services/api/kalvi/api/urls/__init__.py

```diff
@@ -32,5 +32,10 @@
         "reset-password/<str:uid>/<str:token>/",
         UserPasswordResetView.as_view(),
         name="reset-password-through-mail",
+    ),
+    path(
+        "google/",
```

Comment:
```suggestion
        "google/auth/",
```

Reply:
this we will do in next chore work. when we will integrate with front end. (GitHub login URLs also same so i need to change both simultaneously )